gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 allureKotlin = "2.4.0"
-androidGradlePlugin = "8.4.2"
+androidGradlePlugin = "8.8.0"
```

Comment:
ℹ️ `stream-android-push` is using `8.6.1` with Kotlin `2.0.20`

Reply:
I think it should be fine, at the end it is used for compilation process, shouldn't influe how other SDKs were compiled